### ISSUE TITLE
Removing Steem Dollars

### ISSUE THREAD

**Issue by IntinteDAO** [steemit/steem#3554]
Steem Dollar's assumptions were interesting, but the market verified them. Unfortunately, SBD is dead. On the exchanges there is practically no volume of it, it is not even worth about 1 dollar. In addition, users are not rewarded with it (unless?) (or in most cases Steem).

My suggestion is to "delete" Steem Dollar. Of course not directly, but:
- Removal of total earnings in SBD (if there are any more)
- Enabling SBD -> Steem conversion, but not vice versa
- Removal of the Steemit market (it can be replaced by SMT exchange).
- ??

**Comment by mvandeberg** [comment]
> Removal of total earnings in SBD

I am in favor of this purely for the UX.

> Enabling SBD -> Steem conversion, but not vice versa

This has existed on chain since day 1 of Steem.

> Removal of the Steemit market (it can be replaced by SMT exchange).

Removal of SBD has been discusses in the past and has faced significant resistance. Currently, even if we were to implement this, I do not believe it would have much buy in at all and would not be accepted. Right now, I cannot take action on this issue other than the removal of earnings in SBD, but that is already a non-consensus field. I would rather interfaces make the decision to no longer display rewards in SBD, but only in STEEM.

**Comment by IntinteDAO** [comment]
@mvandeberg 

Of course, that's my proposition. Steem Dollar didn't work out, that's all. It is a very susceptible to speculation (which should not be the case, more precisely the price should be stable).

Steem dollar unfortunately disappointed shopkeepers (both), they accepted tokens believing that SBD will be stable, it is not and they lost a lot of money.

He's also a problem with Steem's adoption. I think you'll agree that in order to use Steem, someone needs to undergo terrifying combat training. You have to explain what SBD is about, what Steem is, what Steem Power is. One of the many design problems can be eliminated.

SBD is also an indicator for speculators. If the SBD price doesn't stick to the set price = Something is wrong with Steem (What is true and you will confirm it).

Of course, I realize that the chances of removing Steem Dollar are critically low. But let it be in Issue as a piece of information that someone tried to fix Steem.

**Comment by mvandeberg** [comment]
I will leave open for discussion.

**Comment by KINGdotNET** [comment]
The SBD is not helping and confusing to some. Let's concentrate of promoting one and only STEEM to the market.

**Comment by dnotestein** [comment]
I'm vehemently opposed to removing SBD, a somewhat stable coin is more useful than no stable coin at all, when compared to the huge volatility of existing cryptocurrencies. I wouldn't mind seeing improvements to the pegging mechanism, but that would be a separate issue.

**Comment by pfunks** [comment]
It takes a whole lot less effort to fix SBD than to remove it. Furthermore, (a presumably fixed) SBD is still one of the best things Steem has going for it. Unless you think Steem needs to stay a small, insignificant network where its users should continually cash out, instead of build an economy around. 

This is still possible and frankly, there's no reason not to shoot for the moon at this point: https://steemit.com/steem/@dan/steemit-s-evil-plan-for-cryptocurrency-world-domination

**Comment by iamsmooth** [comment]
> It is a very susceptible to speculation (which should not be the case, more precisely the price should be stable)

It mostly isn't the case. In your scenario of a shopkeeper accepting SBD and losing money that is not due to speculation it is due to the programmed mechanism of reducing the peg when the STEEM price (market cap) drops a lot. In fact a shopkeeper accepting STEEM during the same time period would have lost as much or more.

The exception to this was in late 2017 to early 2018 when SBD increased a lot in value. In that case someone who accepted SBD _expecting it to be worth more than $1_ would have lost a lot as it returned the peg. It is quite debatable how to characterize that, but I would call someone who expects SBD to be worth more than $1 to be a speculator, so it was actually speculators who lost a lot of money there. 

It is far from clear this would ever happen again, at least to the same degree. Since mid 2018 there has been _very little_ effect on the price due to speculation. Again, the current and recent weakness is due to programming and the failure of STEEM to retain or grow value, not SBD speculation.

I agree with the above comments about improve vs. remove (the former being a matter for other issues/PRs). That being said, there really isn't that much that can be done to improve it as long as STEEM continues to decline in value at such a rapid pace, and indeed STEEM itself is more in need of fixing IMO, but that too is a matter for elsewhere.

**Comment by IntinteDAO** [comment]
@dnotestein 

> A somewhat stable coin is more useful than no stable coin at all, when compared to the huge volatility of existing cryptocurrencies. 

Then why do we have two cryptocurrencies instead of 10? For example, one for smart contracts, another for DeFi, a third for exchanges between exchanges, a fourth for securing the value of tokens (...).

If Steem Dollar does not work, it should be liquidated.
